main.nf

```diff
@@ -874,6 +879,110 @@ process trim_galore {
     }
 }
 
+/*
+ * STEP 2+ - SortMeRNA - remove rRNA sequences on request
+ */
+if (!params.remove_rRNA){
+    trimgalore_reads
+        .into { trimmed_reads_alignment; trimmed_reads_salmon }
+} else {
+    sortmerna_fasta = Channel.fromPath([
+        'https://raw.githubusercontent.com/biocore/sortmerna/master/rRNA_databases/rfam-5.8s-database-id98.fasta',
```

Comment:
Is this something that could be made configurable apart form having this hardcoded in the pipeline code? E.g. a separate list of file paths that can be configured by the users? Can be default to this, but the way its currently handled, users can't configure this easily having to wait for new releases of the pipeline....

Reply:
Working on it

Reply:
Done in commit 20ab672304c7ac4662ded9971443709506566bd6
Disadvantage: an empty file / wrong paths in the file are not detected before the execution

Reply:
Resolved, added file verification in commit bfd8803ed3ee5c11ed081b02b0561470ae25f749